src/components/Input/Input.js

```diff
@@ -57,9 +57,11 @@ const Input = forwardRef(
             aria-labelledby={label ? labelId : null}
             ref={ref}
           />
-          <FieldErrorText role="alert" aria-invalid={errorMessage !== null}>
-            {errorMessage}
-          </FieldErrorText>
+          {errorMessage !== null && (
```

Comment:
Can we pass only `errorMessage` instead of `errorMessage !== null` ?

Reply:
I'm trying to keep the previously set condition intact. Not sure why it was like that previously.

Reply:
Okay got it.